hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -1135,28 +1118,55 @@ private CompactionNode addNodeToDAG(String file, String snapshotID,
    *                   arbitrary String as long as it helps debugging.
    * @param seqNum DB transaction sequence number.
    */
-  private void populateCompactionDAG(List<String> inputFiles,
-      List<String> outputFiles, String snapshotId, long seqNum) {
+  private void populateCompactionDAG(List<CompactionFileInfo> inputFiles,
+                                     List<CompactionFileInfo> outputFiles,
+                                     String snapshotId,
```

Comment:
This is not being used anywhere.

Reply:
For now it is here because it is passed to `addNodeToDAG`. But it can be removed.

Will create a follow up task for it.

Reply:
https://issues.apache.org/jira/browse/HDDS-9382